app/controllers/add-job-controller.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const router = new express.Router({ mergeParams: true });
+const Jobs = require('../models/jobs-model');
+const AddJobViewModel = require('./add-job-view-model');
+const progression = require('../models/progression');
+const i18n = require('i18n');
+const Joi = require('joi');
+const celebrate = require('celebrate');
+const validatorSchema = require('./validator-schema');
+/* eslint-disable no-underscore-dangle */
+
+const validator = {
+  post: celebrate({
+    params: {
+      accountId: validatorSchema.accountId.required(),
+    },
+    body: Joi.object().keys({
+      title: Joi.string().allow(''),
+      employer: Joi.string().allow(''),
+      sourceUrl: Joi.string().allow(''),
+      sourceType: Joi.any().valid(['inPerson', 'online']),
```

Comment:
do it ... think the less to have to worry about in each of the controllers the better.... made me even wonder if the const validator in each controller could be relocated as well and imported.  But at the same time think its fine where it is because you can see at a glance what is being validated.